overlays/nixpkgs.nix

```
@@ -105,7 +105,7 @@ final: prev:
         [
           # Patch No. 1 needs a rebase on the new version, fetch the rebased version from the upstream PR.
           (final.fetchpatch {
-            url = "https://raw.githubusercontent.com/katexochen/nixpkgs/refs/heads/libnvidia-container/1.17.8/pkgs/by-name/li/libnvidia-container/0001-ldcache-don-t-use-ldcache.patch";
+            url = "https://raw.githubusercontent.com/NixOS/nixpkgs/233e50594e942b36850f6cdc6129d60e46b4f45d/pkgs/by-name/li/libnvidia-container/0001-ldcache-don-t-use-ldcache.patch";
```

```suggestion
            url = "https://raw.githubusercontent.com/NixOS/nixpkgs/0c909a5522020455d518a7f028e4850d55080bf6/pkgs/by-name/li/libnvidia-container/0001-ldcache-don-t-use-ldcache.patch";
```
You meant to link to your patch, right? https://github.com/NixOS/nixpkgs/commit/0c909a5522020455d518a7f028e4850d55080bf6

---

well, I was lazy and used latest master commit as the file is the same, but using the actual commit is more meaningful of cause.